CONTRIBUTING.md

```diff
@@ -0,0 +1,121 @@
+# Contributing
+
+When contributing to this repository as a curriculum developer, please first
+discuss the change you wish to make via issue, email, or any other method with
+James Cranston before create a feature request branch. 
+
+Please note we have a code of conduct, please follow it in all your
+interactions with the project.
+
+## Merge Request Process for Curriculum Contributions
+
+1. Please create a feature branch for your update in the following format:
+    ```
+    <github username>/contributing/<short description>
+    ```
+    As an example:
+    ```
+    jcranston/contributing/update-project-documentation
+    ```
+   Ensuring branches are named in this format will ensure they do not get
```

Comment:
Suggest revising to "Naming branches in this format will ensure..." to avoid using ensure twice so close together.